### ISSUE TITLE
Default Image Streams and Templates path to update is not correct

### ISSUE THREAD

**Issue by nak3** [openshift/openshift-docs#1853]
Chap. [Performing Manual Cluster Upgrades - Updating the Default Image Streams and Templates](https://docs.openshift.com/enterprise/3.1/install_config/upgrading/manual_upgrades.html#updating-the-default-image-streams-and-templates)

```
# oc create -n openshift -f /usr/share/openshift/examples/image-streams/image-streams-rhel7.json
# oc create -n openshift -f /usr/share/openshift/examples/db-templates
# oc create -n openshift -f /usr/share/openshift/examples/quickstart-templates
...
```

Should be:

```
# oc create -n openshift -f /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/<VERSION>/....
```

**Comment by adellape** [comment]
@nak3 Both locations exist. `/usr/share/openshift/examples/` is supposed to get updated whenever the installer is run (first time installations and upgrades). `/usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples` is updated whenever the installer RPMs are updated.

A `latest` symlink was added to the latter location in https://github.com/openshift/openshift-ansible/pull/1519. That's not yet in the 3.1 version of the installer packages yet, but might be in the advisory that's about to ship.

I've found inconsistent results in my testing (mostly trying `v3_1_minor` upgrades), as far as when files in one or the other location are updated, but my hosts may be bad examples. I'd prefer to use the currently-documented location (`/usr/share/openshift/examples/`) if we can rely on it. It's shorter and hopefully folks are using the upgrader to apply errata.

If we went with the latter, we could reference the `latest` directory (after the aforementioned advisory is shipped) instead of `<VERSION>`, as suggested. There are times when we ship updates to the installer packages on there own, and don't require the admin to do a full OSE upgrade, I suppose. So in that scenario, it seems like that `latest` directory would be providing then with the actual latest files they have on disk.

cc @tpoitras @sdodson

**Comment by nak3** [comment]
> /usr/share/openshift/examples/ is supposed to get updated whenever the installer is run (first time installations and upgrades).

Yes, after the installer is run, `/usr/share/openshift/examples/` is supposed to get updated.
But [this section](https://docs.openshift.com/enterprise/3.1/install_config/upgrading/manual_upgrades.html#updating-the-default-image-streams-and-templates) is about **manual upgrades** and there are no commands like `atomic-openshift-installer upgrade` or `ansible-playbook .. upgrade.yml`.

Although I agree with your comments if it is the section [automated upgrades](https://docs.openshift.com/enterprise/3.1/install_config/upgrading/automated_upgrades.html), I think [this section](https://docs.openshift.com/enterprise/3.1/install_config/upgrading/manual_upgrades.html#updating-the-default-image-streams-and-templates) should be changed.

**Comment by mburke5678** [comment]
@nak3 @adellape Is this still an issue that needs to be addressed? Or, can we close?

**Comment by nak3** [comment]
Actually, 3.1 docs did not fix this issue, but 3.5 fixed it. So, let's close. Nobody will update to 3.1.
